lib/checkuninitvar.cpp

```diff
@@ -196,6 +196,8 @@ void CheckUninitVar::checkScope(const Scope* scope, const std::set<std::string>
 void CheckUninitVar::checkStruct(const Token *tok, const Variable &structvar)
 {
     const Token *typeToken = structvar.typeStartToken();
+    while (typeToken->next() && typeToken->next()->str() == "::")
```

Comment:
I would prefer a more explicit check here `Token::Match(typeToken, "%name% ::")` that extra isName() should be very quick.